Dockerfile

```diff
@@ -3,17 +3,12 @@ FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.10 AS
 
 WORKDIR /workspace
 
-# Copy the Go Modules manifests
-COPY go.mod go.mod
-COPY go.sum go.sum
-# cache deps before building and copying source so that we don't need to re-download as much
-# and so that source changes don't invalidate our downloaded layer
+COPY ./ ./
 RUN go mod download
+# needed for docker build but not for local builds
+RUN go mod vendor
 
-# Copy the go source
-COPY main.go main.go
-COPY api/ api/
-COPY controllers/ controllers/
+RUN make generate
 
 # Build
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -mod=mod -o manager main.go
```

Comment:
I'd prefer replacing this with simply `RUN make build` so that we don't have two places to maintain where the `manager` binary is built.

Note that doing so also requires adjusting the `COPY` command below to `COPY --from=builder /workspace/bin/manager .` This would also let us get rid of the `RUN make generate` layer above since `generate` is a prerequisite of `build`.

Reply:
Yes, you are correct.  I will fix this.